packages/components/src/components/post-linkarea/post-linkarea.scss

```
@@ -0,0 +1,4 @@
+:host {
+  display: block;
```

Since you're not sure whether the element inside `post-linkarea` will be an inline or block element, it's better to set the wrapper to `display: contents`. This makes the `post-linkarea` behave as if it were replaced by its content in the DOM. Its logic remains intact, but it no longer acts as a box. Here’s an article that explains this well: https://la-cascade.io/articles/comment-fonctionne-css-display-content.
```suggestion
  display: contents;
```